src/services/comment.service.js

```diff
@@ -13,6 +13,40 @@ class CommentService {
   static createComment(newComment) {
     return Comments.create(newComment);
   }
+
+  /**
+   * @param {number} property
+   * @returns {find} this function finds comment
+  */
+  static findCommentByProperty(property) {
+    return Comments.findOne({
+      where: property
+    });
+  }
+
+  /**
+   * @param {number} property
+   * @returns {delete} this function deletes comment
+  */
+  static deleteCommentByProperty(property) {
+    return Comments.destroy({
+      where: property
+    });
+  }
+
+  /**
+   *
+   *
+   * @static
+   * @param {req} commentId
+   * @memberof CommentController
+   * @returns {object} this function deletes comment on posted on thread
+   */
+  static async deleteCommentBySubjectType(commentId) {
```

Comment:
`deleteCommentBySubjectType` does not say exactly the subjectTrip you are deleting.

Reply:
I named it like this to be reusable when the delete accommodation wants to use it, then they can pass subjectType to `accommodation` ?

Reply:
On line 46, `subjectType` is already hardcoded. Also, I don't see any difference between this method and line 31. You should use that instead of this